lib/model/emoji.dart

```diff
@@ -116,6 +116,13 @@ final class EmojiCandidate {
 
 /// The portion of [PerAccountStore] describing what emoji exist.
 mixin EmojiStore {
+  /// An [EmojiDisplay] for the specified emoji.
+  ///
+  /// Use [EmojiDisplay.resolve] on the result to apply the user's [Emojiset]
+  /// setting.
+  ///
+  /// May be a [TextEmojiDisplay] even if the emojiset is not [Emojiset.text];
+  /// this happens for image emojis whose URLs don't parse.
   EmojiDisplay emojiDisplayFor({
```

Comment:
Looking at the implementation, there are a few other cases where this happens, right? Seems like more generally it's when we can't understand the data that describes the emoji.